sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -69,7 +69,7 @@ abstract class StreamExecution(
     override val sparkSession: SparkSession,
     override val name: String,
     private val checkpointRoot: String,
-    analyzedPlan: LogicalPlan,
+    val analyzedPlan: LogicalPlan,
```

Comment:
where do we call it outside of `StreamExecution`?

Reply:
Here we removed `analyzedPlan` in the subclass params: https://github.com/apache/spark/pull/31842/files#diff-cf2dc65d70aa7bc490d237f1c50f5fdf51e244fabed782977a94d2934c208551L41. Now the `analyzedPlan` is called directly in subclass `MicroBatchExecution` and `ContinuousExecution`.